src/main/util/bindContextMenu.js

```diff
@@ -0,0 +1,49 @@
+import { Menu } from 'electron';
+
+export default function bindContextMenu(browserWindow) {
+  browserWindow.webContents.on('context-menu', (event, params) => {
+    const template = [];
+    const { isEditable, editFlags } = params;
+
+    if (isEditable) {
+      template.push({
+        label: 'Undo',
+        role: editFlags.canUndo ? 'undo' : '',
+        enabled: editFlags.canUndo
+      }, {
+        label: 'Redo',
+        role: editFlags.canRedo ? 'redo' : '',
+        enabled: editFlags.canRedo
+      }, {
+        type: 'separator'
+      }, {
+        label: 'Cut',
+        role: editFlags.canCut ? 'cut' : '',
+        enabled: editFlags.canCut
+      }, {
+        label: 'Copy',
+        role: editFlags.canCopy ? 'copy' : '',
+        enabled: editFlags.canCopy
+      }, {
+        label: 'Paste',
+        role: editFlags.canPaste ? 'paste' : '',
+        enabled: editFlags.canPaste
+      }, {
+        label: 'Paste and Match Style',
+        role: editFlags.canPaste ? 'pasteandmatchstyle' : '',
+        enabled: editFlags.canPaste
+      }, {
+        label: 'Select All',
+        role: editFlags.canSelectAll ? 'selectall' : '',
+        enabled: editFlags.canSelectAll
+      });
+    }
+
```

Comment:
is there a possibility to show a "paste and go" when clicking on the address bar? rest works fine!

Reply:
It's also quite impossible to cut text in the address bar :P I could address this in another PR

Reply:
I'm hesitant on the "Paste and Go" idea, just because it falls completely outside the realm of standard UX for this type of interface.  It would also introduce some complexity because we would need a way to know it's the address bar from here, and I'm not sure how we could figure that out easily.

Reply:
> It's also quite impossible to cut text in the address bar

Not sure what you mean by this.  Since you say we can address separately, maybe that makes sense. :)

Reply:
Chrome has this too, but I agree on the complexity increase :P It could be something for the future.
The 2nd comment is a note to myself. When you copy a piece of text in the URL bar (not all of it) and try to cut it, the selection is gone. This is unrelated to the PR 😄